src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -155,20 +173,22 @@ export class QuickBar extends LitElement {
   ): CommandItem[] {
     return items
       .filter(({ text }) =>
-        fuzzySequentialMatch(filter.toLowerCase(), text.toLowerCase())
+        fuzzySequentialMatch(filter.toLowerCase(), [text.toLowerCase()])
       )
       .sort((itemA, itemB) => compare(itemA.text, itemB.text));
   }
 
-  private _filterEntityItems(
-    entity_ids: HassEntity["entity_id"][],
-    filter: string
-  ): HassEntity["entity_id"][] {
-    return entity_ids
-      .filter((entity_id) =>
-        fuzzySequentialMatch(filter.toLowerCase(), entity_id)
+  private _filterEntityItems(filter: string): HassEntity[] {
+    return this._entities
+      .filter(({ entity_id, attributes: { friendly_name } }) =>
+        fuzzySequentialMatch(filter.toLowerCase(), [
+          entity_id,
+          ...(friendly_name ? [friendly_name] : []),
+        ])
```

Comment:
```suggestion
      .filter(({ entity_id, attributes: { friendly_name } }) => {
      	const values = [entity_id];
        if (friendly_name) {
        	values.push(friendly_name);
        }
        return fuzzySequentialMatch(filter.toLowerCase(), values);}
```

![image](https://user-images.githubusercontent.com/5662298/95846586-e4f67f00-0d4b-11eb-9a4f-e05e8fd84126.png)